js/classes/room.js

```diff
@@ -1,4 +1,12 @@
-const rng = new Math.seedrandom();
+// TODO: this is not a class yet but I plan to convert it to one. -kc 8/6/2020
+
+export const enterNewRoom = (iDir = null) => {
+  currentRoom = iDir !== null ? currentRoom.newRooms.rooms[iDir] : generateRoom();
+  currentRoom.newRooms = generateNewPaths();
+  console.log(
```

Comment:
Is the game going to be ran from the console or is it just lacking the UI for right now?

Reply:
I'm making the UI. The console logs are probably just to let us know something is working.